scripts/linters/test_files/invalid_ts_ignore.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Invalid syntax .ts file, used by scripts/linters/
+ * js_ts_linter_test.py. Ts ignore is used but no comment explains
+ * the ts-ignore.
+ */
+
+let a: string;
+
+// @ts-ignore
```

Comment:
Add more cases, e.g. where there's a comment but not of the correct format. (Try to signal, through other comments in this test file, what each of those cases checks.)

Reply:
done